app/routines/newflow/educator_signup/activate_educator.rb

```diff
@@ -4,6 +4,9 @@
 module Newflow
   module EducatorSignup
     class ActivateEducator
+
+      SECURITY_LOG_EVENT_TYPE = :user_became_activated
```

Comment:
Can this constant be in one place? It is in the educator and student signup flows.

Reply:
There is very little code that is different in this class and the Activate Student class. Can there be a base class with the shared code?

Reply:
Fixed.